packages/lesswrong/components/ea-forum/users/EAGApplicationImportForm.tsx

```diff
@@ -487,23 +492,25 @@ const EAGApplicationImportForm = ({classes}: {
       We've found your application data from {event}.
     </Typography>
     
-    <div className={classes.callout}>
-      <Typography variant="body1" className={classes.overwriteText}>
-        Would you like to overwrite your EA Forum profile data?
-      </Typography>
-      <div>
-        <button type="submit"
-          onClick={(e) => handleSubmit(e, true)}
-          className={classes.submitBtn}
-          {...submitLoading ? {disabled: true} : {}}
-        >
-          Overwrite and Submit
-        </button>
-        <Typography variant="body1" className={classes.overwriteBtnAltText}>
-          or see details and make changes below
+    <AnalyticsContext pageSectionContext="overwriteCallout">
```

Comment:
I don't know off the top of my head that button submit captures events, but I assume you've tested.